src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -517,6 +506,7 @@ FixedwingPositionControl::tecs_status_publish(float alt_sp, float equivalent_air
 	tecs_status.throttle_sp = _tecs.get_throttle_setpoint();
 	tecs_status.pitch_sp_rad = _tecs.get_pitch_setpoint();
 	tecs_status.throttle_trim = throttle_trim;
+	tecs_status.underspeed_mode_enabled = _tecs.underspeed_detected();
```

Comment:
nitpick - not a strong opinion - but we could also just output the percentage undersped directly from tecs, and also then report it in tecs status, as the behavior is ramped in continuously.

Reply:
Good idea. I've added that.